pubspec.yaml

```diff
@@ -1,15 +1,11 @@
-name: package_config
-version: 2.0.0
+name: package_config_2
```

Comment:
This would require tests to be updated too, right?

Reply:
It should match, I forgot to change this one back after running tests.